### PR TITLE
Delete the nll-facts directory to avoid running out of hard drive space.

### LINKED ISSUE #412: When running tests delete logs and NLL facts after each test
Currently, sometimes CI crashes with the following error:
```
thread 'rustc' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 112, kind: Other, message: "There is not enough space on the disk." }', compiler\rustc_mir\src\borrow_check\nll.rs:280:62
```

https://github.com/viperproject/prusti-dev/pull/405/checks?check_run_id=1902692039

### PR DESCRIPTION
**Pull request by vakaras**
An attempt to fix #412

### CODE REVIEW

File: prusti-tests/tests/compiletest.rs [hunk 1]
```diff
@@ -107,6 +107,10 @@ fn run_prusti_tests(group_name: &str, filter: &Option<String>, rustc_flags: Opti
         config.src_base = path;
         run_tests(&config);
     }
+
+    // Delete the nll-facts directory to avoid running out of hard drive
+    // space.
+    std::fs::remove_dir_all("nll-facts").unwrap();
```
Comment (fpoli) [inline prusti-tests/tests/compiletest.rs]: I think the default location is `log/nll-facts`.

Comment (vakaras) [reply prusti-tests/tests/compiletest.rs]: Thanks! Let's see whether this helps.

**Comment by fpoli** [comment]
`parse` and `typecheck` tests don't generate that folder.

**Comment by vakaras** [comment]
@kammola Could you please try to rebase your PR on the current master?

**Comment by kammola** [comment]
> @kammola Could you please try to rebase your PR on the current master?

It still gives the same error

**Comment by vakaras** [comment]
> > @kammola Could you please try to rebase your PR on the current master?
> 
> It still gives the same error

That is bad. I need to investigate why this happens. For now, please leave the PR as it is.

**Pull request merged**